setup/ansible-inventory

```diff
@@ -77,10 +77,6 @@ iojs-release-armv7-wheezy-1
 release-scaleway-debian7-arm-1
 
 [iojs-raspbian]
-test-requireio_joeyvandijk-debian7-arm_pi2-1
-test-requireio_svincent-debian7-arm_pi2-2
```

Comment:
What happened to these two Pis?

Sorry if you covered it in your update, didn't see it 😁 .